bullet_train-api/test/factories/teams.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+  factory :team, class: Team do
+    name { "Test Team" }
+    slug { "test_team" }
+    time_zone { "Pacific Time (US & Canada)" }
+  end
+
+  factory :team_example, class: Team do
+    id { 42000 }
```

Comment:
FYI @jagthedrummer, we hardcode the team ID for doc gen so all child records have the same team_id reference.